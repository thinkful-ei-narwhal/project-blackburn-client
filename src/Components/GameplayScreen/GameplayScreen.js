import React, { Component } from "react";
import gameplayImg from "./../../Images/DetectiveAtDesk.jpg";

class GameplayScreen extends Component {
  //todos:
  //Show the gameplay screenshot

  componentDidMount() {}

  render() {
    return (
      <div>
        <img src={gameplayImg} height="400" width="300" alt="Gameplay image" />
      </div>
    );
  }
}

export default GameplayScreen;
