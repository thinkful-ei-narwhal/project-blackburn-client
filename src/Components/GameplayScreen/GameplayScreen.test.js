import React from 'react';
import { shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import GameplayScreen from './GameplayScreen';

describe(`Adds gameplay img to dom without crashing`, () => {
    it('renders without crashing', () => {
        const wrapper = shallow(
            <GameplayScreen />
    )
    .find('img')
    expect(toJson(wrapper)).toMatchSnapshot()
    })
})