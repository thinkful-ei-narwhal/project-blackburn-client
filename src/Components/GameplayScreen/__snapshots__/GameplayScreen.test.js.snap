// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Adds dashboard to dom without crashing renders without crashing 1`] = `
<img
  alt="Gameplay"
  height="400"
  src="DetectiveAtDesk.jpg"
  width="300"
/>
`;

exports[`Adds gameplay img to dom without crashing renders without crashing 1`] = `
<img
  alt="Gameplay"
  height="400"
  src="DetectiveAtDesk.jpg"
  width="300"
/>
`;
